server/serverMethods.js

```diff
@@ -760,12 +711,6 @@ function ServerMethods(aLogLevel, aModules) {
         if (!dialedNumberInfo || dialedNumberInfo.googleIdToken !== googleIdToken) {
           return aRes.status(400).send(new ErrorInfo(400, 'Unknown phone number.'));
         }
-        if (dialedNumberInfo.uuid) {
-          var params = {
-            call_uuid: dialedNumberInfo.uuid,
-          };
-          plivoClient.hangup_call(params);
```

Comment:
If you click the hang-up button before the callee answers the phone, the call keeps dialing. Is there a way to disconnect (or hang up) the SIP call if the callee hasn't answered? (Calling force disconnect on the SIP connection doesn't hang up the call.)

Reply:
One other thing, @msach22: When I run `npm install` on this branch, I see a lot of diffs in the package-lock.json file.

Reply:
@jeffswartz updated the package-lock.json file!